src/tunnel.js

```diff
@@ -1,35 +1,36 @@
-var ngrokLauncher = require('screener-ngrok');
-var Promise = require('bluebird');
-var url = require('url');
-var sauceConnectLauncher = require('sauce-connect-launcher');
-var sauceConnection;
+const ngrokLauncher = require('screener-ngrok');
+const Promise = require('bluebird');
+const url = require('url');
+const Saucelabs = require('saucelabs').default;
```

Comment:
Question: Sauce Connect used to be pinned to a specific version (4.6.2 currently). Does `saucelabs` package pin it? If not, can it (or should it) be pinned? 

We currently have the specific version listed in the README and Screener Docs - these docs will need to be updated or removed

Reply:
Default is now 4.6.2 added environment variable SAUCE_CONNECT_VERSION to be able to configure